components/org.wso2.identity.webhook.wso2.event.handler/src/main/java/org/wso2/identity/webhook/wso2/event/handler/api/builder/WSO2UserOperationEventPayloadBuilder.java

```diff
@@ -234,4 +301,44 @@ private Group buildGroup(Map<String, Object> properties, AbstractUserStoreManage
 
         return group;
     }
+
+    public enum RecoveryCredentialType {
+        PASSWORD, USERNAME, NONE
+    }
```

Comment:
what is this recovery credential type and why username comes here

Reply:
Fixed